verify-service/src/main/java/io/inji/verify/controller/VPResultController.java

```diff
@@ -46,6 +47,9 @@ public ResponseEntity<Object> getVPResult(@PathVariable String transactionId, Ht
             } catch (VPSubmissionNotFoundException e) {
                 log.error(e.getMessage());
                 return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new ErrorDto(ErrorCode.NO_VP_SUBMISSION));
+            } catch (InvalidVpTokenException e) {
```

Comment:
@sarupmoni why are you throwing status 404 exception for invalid VP token, it should be status 200 with verification results as failed